server/migrations/20210225221835-add-clover-finance.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.bulkInsert('Chains', [{
```

Comment:
Need to make sure you also set a `base` here of `substrate`. Note that after the Darwinia migration was created, there is also `20210217161948-add-chain-base-and-ss58-prefix.js` which adds that field for Darwinia.